ragaai_catalyst/tracers/agentic_tracing/upload/upload_trace_zip_automation.py

```
@@ -0,0 +1,582 @@
+        }
+        payload = json.dumps({
+            "datasetName": dataset_name,
+            "metrics": metrics
+        })
+        response = requests.request("POST",
+                                    f"{RagaAICatalyst.BASE_URL}/v1/llm/trace/metrics",
+                                    headers=headers,
+                                    data=payload,
+                                    timeout=10)
+        if response.status_code != 200:
+            raise ValueError(f"Error inserting agentic trace metrics")
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)